modules/admin_manual/pages/enterprise/collaboration/collabora_online_integration.adoc

```diff
@@ -0,0 +1,64 @@
+= Secure View
+
+When Collabora Online is properly setup and integrated into ownCloud Server (10.2 or above) Secure View functionality is available.
+Secure View is a mode where users can place limitations on files and folders that are shared. 
+These limitations include:
+[[secure-view-mode-limitations]]
+* No copying
+* No downloading
+* No editing
+* Watermarking with optional print and export
++
+NOTE: Administrators can configure the watermark text.
+* Printing and exporting to PDF with watermarks included
```

Comment:
Should be merged with the last point above, I guess?!

Reply:
Do you mean with the list of four limitations? It's merged, in a way, in that the admonition is part of the previous bullet point, as you can see in the screenshot below.

![admonition-in-list](https://user-images.githubusercontent.com/196801/61455882-04de9d80-a965-11e9-8075-c2fb9302e435.png)

Reply:
"Watermarking with optional print and export" and "Printing and exporting to PDF with watermarks included" is exactly the same :) would just remove "with optional print and export" from the bullet point before the note.

Reply:
Just applied your feedback.

Reply:
Please add a last bullet point: "Optional printing and exporting to PDF with watermarks included"

Reply:
Okay. Misunderstood previously.